sky/jobs/utils.py

```diff
@@ -806,7 +806,10 @@ def stream_logs(cls,
 
         from sky.skylet import job_lib, log_lib
         from sky.skylet import constants
-        from sky.jobs.utils import stream_logs_by_id
+        try:
+            from sky.jobs.utils import stream_logs_by_id
+        except ImportError:
```

Comment:
Should we leave a quick comment to deprecate this in v0.8.0?

Reply:
Added. Thanks!